proposals/description/repay_fuse_bad_debt.ts

```diff
@@ -19,6 +19,13 @@ export default fip_x;*/
 const fip_x: ProposalDescription = {
   title: 'Repay Fuse Bad Debt',
   commands: [
+    {
+      target: 'pcvGuardianNew',
+      values: '0',
+      method: 'setSafeAddress(address)',
+      arguments: ['{fuseFixer}'],
```

Comment:
If we're making this a safe address, think we should add a generic `withdrawERC20()` function to. In case funds are accidentally sent to this address, otherwise we could trap them